fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1806,7 +1806,7 @@ void buildPartitions(Database db, OlapTable table, List<PhysicalPartition> parti
 
         TabletTaskExecutor.CreateTabletOption option = new TabletTaskExecutor.CreateTabletOption();
         option.setEnableTabletCreationOptimization(table.isCloudNativeTableOrMaterializedView()
-                && Config.lake_enable_tablet_creation_optimization);
+                && (Config.lake_enable_tablet_creation_optimization || table.enablePartitionAggregation()));
```

Comment:
Please add comment to explain what is enableTabletCreationOptimization and what is enablePartitionAggregation and what relationship between this option